sites/fast-website/src/public/index.html

```diff
@@ -9,6 +9,24 @@
     <fast-design-system-provider use-defaults>
       <fast-button>Hello, World!</fast-button>
       <gradient-picker>stub</gradient-picker>
+      <fast-section-header
+        short="Features & Utilities"
+        body="At id vitae iaculis volutpat laoreet. Massa ut scelerisque ut ac, in aliquam odio. Semper sagittis blandit in sed turpis lorem at diam iaculis."
+      >
+        <h5 slot="heading">A COMPREHENSIVE TOOLKIT</h5>
```

Comment:
All-uppercase text should be Title Case in the markup and use `text-transform: uppercase`

Reply:
I might go further and suggest not using text transform in CSS at all due to the potential issues with localization in certain languages. Our principle has typically been that we don't transform text and we rely on the string itself.